android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -258,14 +281,98 @@ private void unregisterListener() {
     if(zoomSDK.isInitialized()) {
       MeetingService meetingService = zoomSDK.getMeetingService();
       meetingService.removeListener(this);
+      InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+      inMeetingService.removeListener(this);
+      InMeetingShareController inMeetingShareController = inMeetingService.getInMeetingShareController();
+      inMeetingShareController.removeListener(this);
```

Comment:
I think we should check if `inMeetingService`/`inMeetingShareController` is null before calling `removeListener`

Reply:
You are absolutely right. I missed doing this in a few other places, too. I'll fix that.